packages/elements-core/src/hooks/useComputeMarkdownHeadings.ts

```diff
@@ -1,20 +1,19 @@
 import { MDAST } from '@stoplight/markdown';
 import * as React from 'react';
 import { Parent } from 'unist';
+import { selectAll } from 'unist-util-select';
 
 import { IArticleHeading } from '../types';
 
-const selectAll = require('unist-util-select').selectAll;
-
 export function useComputeMarkdownHeadings(tree: MDAST.Root) {
   return React.useMemo(() => computeMarkdownHeadings(tree), [tree]);
 }
 
 export function computeMarkdownHeadings(tree: MDAST.Root): IArticleHeading[] {
-  return selectAll(':root > [type=heading]', tree)
+  return (selectAll(':root > [type=heading]', tree) as MDAST.Heading[])
     .map((heading: MDAST.Heading) => ({
       title: findTitle(heading),
-      id: heading.data && (heading.data.id as string | undefined),
+      id: (heading.data?.id as string | undefined) || '',
```

Comment:
What is this change? Is this related?

Reply:
I've replaced `unist-util-select`, earlier imported by `require` and having no typing. This change was to have some typing instead using `any`